solr/config/schema.xml

```diff
@@ -248,6 +248,9 @@
 
     <!-- exhibits fields -->
     <field name="full_text_search" type="text" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
+    <field name="full_text_search_en" type="text_en" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
+    <field name="full_text_search_pt" type="text_pt" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
+    <field name="full_text_search_id" type="text_id" indexed="true" stored="true" termVectors="true" termPositions="true" termOffsets="true" multiValued="true" />
     <field name="full_text_unstem_search" type="textNoStem" indexed="true" stored="true" multiValued="true" />
```

Comment:
Do we need to create `full_text_unstem_search` fields for each language as well?

Reply:
No, I think that should be unnecessary. I think we only need language-specific fields when we're doing stemming, lemmatization, or language-specific tokenization.